modules/sievefilters/site.js

```diff
@@ -501,13 +492,11 @@ $(function () {
 
         function save_script(imap_account) {
             if ($('.modal_sieve_script_name').val() === "") {
-                $('.sys_messages').html('<span class="err">You must provide a name for your script</span>');
-                Hm_Utils.show_sys_messages();
+                $('.sys_messages').html('<div class="alert alert-danger alert-dismissible fade show" role="alert"><i class="bi bi-exclamation-triangle me-2"></i><span class="err">You must provide a name for your script</span><button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button></div>');
```

Comment:
Can you please refactor so there will be one class to manage alert and to build look according to the type. In that way we won't repeat code.

Reply:
Yes @josaphatim  , you're right, I'm working on refactoring all that